app/models/user.rb

```diff
@@ -1,9 +1,15 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] [REQUIRED] Good job so far! The application asks for `confirmation` of email.
Please check to see  `:confirmable` should be enabled in the user model (`user.rb` file). It look like something similar to what is below:

```ruby
	devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```